### PR TITLE
Add a polling interval in email workflow using polling_interval_seconds

### PR DESCRIPTION
**Pull request by bhavanijayakumaran**
Adding a polling interval to the retry.

https://github.com/Cray/chapel-private/issues/4149

We usually see timeouts if the git server is busy, the default retry polling wait is 1 sec, retrying every second will add more workload to already loaded system so increased the polling interval to 60 sec.

Signed-off-by: bhavanijayakumaran <82669529+bhavanijayakumaran@users.noreply.github.com>

### CODE REVIEW

File: .github/workflows/email.yml [hunk 1]
```diff
@@ -29,9 +29,10 @@ jobs:
         with:
           max_attempts: 4
           retry_on_exit_code: 2
-          timeout_seconds: 60  
+          timeout_seconds: 180
+          polling_interval_seconds: 60  
           command: |
-              curl --location --request GET $COMMIT_URL --header 'X-API-Key: ${{ secrets.GITHUB_TOKEN}}' -o commits.json
+              curl  --location --request GET $COMMIT_URL --header 'X-API-Key: ${{ secrets.GITHUB_TOKEN}}' -o commits.json
```
Comment (lydia-duncan) [inline .github/workflows/email.yml:35]: Was this extra space intentional?

**Review by lydia-duncan — APPROVED** [review]
Looks good minus a potentially minor typo!

File: .github/workflows/email.yml (hunk 1 above)
Comment (bhavanijayakumaran) [reply .github/workflows/email.yml:35]: oops.. accidentally added it.

Comment (bhavanijayakumaran) [reply .github/workflows/email.yml]: fixed the typo

**Pull request merged**